components/browse/ContentCardSection.vue

```diff
@@ -1,12 +1,26 @@
 <template>
   <div
+    v-if="section"
     class="banner"
     data-qa="browse section"
   >
-    <h2>{{ section.fields.headline }}</h2>
-    <p>{{ section.fields.text }}</p>
+    <h2
+      v-if="section.fields"
```

Comment:
There is an if for every element right now, that looks as an unnecessary repetition. I think that you don't have to check if the section exists on line 3, as that happened already in the page itself. Check on line 3 if the section.fields exists.